pkg/client/unversioned/helper.go

```diff
@@ -228,9 +228,13 @@ func NegotiateVersion(client *Client, c *Config, version string, clientRegistere
 		if serverVersions.Has(clientVersion) {
 			// Version was not explicitly requested in command config (--api-version).
 			// Ok to fall back to a supported version with a warning.
-			if len(version) != 0 {
-				glog.Warningf("Server does not support API version '%s'. Falling back to '%s'.", version, clientVersion)
-			}
+			// TODO: caesarxuchao: temporary disable this warning message.
```

Comment:
TODO should be to enable this again.
I will recommend filling an issue and referencing it here.